src/WebApp/wwwroot/js/questions.js

```diff
@@ -0,0 +1,10 @@
+﻿window.addEventListener("load", function () {
```

Comment:
Maybe we should put this function to top level and give it a name, so that it can be applied to a class `.live-date`, which can then be updated later live by calling the named function repeatedly (for a more meaningful "x years ago" date).

Reply:
(Seeing that you have already done this in a commit: It's probably a better idea to keep the UTC date in a data-argument, because otherwise we loose our original date on an update. So the code should get the date from the [data-date] and then transform it and write it into innerHTML.

Reply:
https://github.com/codidact/core/pull/88/commits/1dbfba00f64ac76ebe4d6539ca2cb5e6c017e944